packages/core/src/util/iitmHook.js

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
question: why is the iitmhook & requirehook in core/src/util and the hook.js in core/src/tracing?

Reply:
Moved `hook.js` to the core/src/util